mempool/clist_mempool.go

```diff
@@ -152,6 +152,9 @@ func WithMetrics(metrics *Metrics) CListMempoolOption {
 
 // Safe for concurrent use by multiple goroutines.
 func (mem *CListMempool) Lock() {
+	if mem.recheck.setRecheckFull() {
+		mem.logger.Debug("the state of recheckFull has flipped")
```

Comment:
I think this message is not very useful if it doesn't say from X to Y, as we need to trace back all occurrences of this message for a node to find out if it is now full, or no longer full.
I think this message would be better inside `recheck.setRecheckFull` where it can provide this info.

In any case, I don't know if it should be done here, or in a different PR from `main`, and then backported all the way down.

Reply:
Agree. I will do it from main in a different PR.